adb_graphics/default_specs.yml

```diff
@@ -1547,6 +1562,42 @@ wspeed: # Wind Speed
         field2: VGRD_P0_L103_{grid}
     unit: kt
     wind: True
+  5mb:
```

Comment:
Perhaps it would be helpful to add an anchor here that can be referenced in the other new levels below? This would reduce duplication.

```
5mb: &contoured_ua_wspeed
...
10mb:
  <<: *contoured_ua_wspeed
  title: 10mb Wind
...
```

Reply:
I'm not sure why the changes aren't showing up, but I made this change, though I used "shaded_ua_wspeed" instead of "contoured_ua_wspeed".